app/calculation/convert-currency.js

```diff
@@ -1,12 +1,12 @@
-function convertStringToPence (valueInPence) {
-  const currencyArray = valueInPence.split('.')
+function convertStringToPence (valueInPounds) {
+  const currencyArray = valueInPounds.split('.')
   const pounds = currencyArray[0]
   const pence = (currencyArray[1] || '00').padEnd(2, '0')
   return Number(pounds + pence)
 }
 
-function convertToPounds (valueInPounds) {
-  return valueInPounds / 100
+function convertToPounds (valueInPence) {
+  return valueInPence / 100
```

Comment:
These variable names were the wrong way around.  No functionality change, just corrected naming so they make sense.